.env.example

```diff
@@ -1,6 +1,6 @@
 DB_HOST='localhost'
 DB_NAME='EffektDonasjonDB_Local'
-DB_USER='root'
+DB_USER='myuser'
```

Comment:
I think the current `README.md` documentation guides the user into using `root` as user when setting up their `mysql` environment. Maybe keep `root` here, or make the documentation agnostic about which username to use?

Reply:
As far as I can tell, a `root` user is always created, so having `DB_USER='root'` doesn't actually change anything. 

Will take a look if something should be updated in readme :+1:

Reply:
Actually all those readme command using root still work. Perhaps some should be swapped to `myuser` but not fully clear as :norway: is still using root. I'll leave it as it is for now